ledger/participant-integration-api/src/main/scala/platform/store/backend/common/EventStorageBackendTemplate.scala

```diff
@@ -28,8 +28,7 @@ abstract class EventStorageBackendTemplate(
     queryStrategy: QueryStrategy,
     ledgerEndCache: LedgerEndCache,
     stringInterning: StringInterning,
-    // TODO Refactoring: This method is needed in pruneEvents, but belongs to [[ParameterStorageBackend]].
-    //                   Remove with the break-out of pruneEvents.
+    // This method is needed in pruneEvents, but belongs to [[ParameterStorageBackend]].
```

Comment:
this will be reworked in upcoming epic where these concerns will surface differently. kept the incentive without TODO